test/integration/targets/nxos_snmp_contact/tests/nxapi/sanity.yaml

```diff
@@ -0,0 +1,4 @@
+---
```

Comment:
same like above. This file is not necessary as the logic is handled in tasks/. This will run nxapi tests twice.